bin/run-l10n-extraction

```diff
@@ -28,74 +28,68 @@ error() {
   exit 1
 }
 
-run_l10n_extraction() {
```

Comment:
This function was fine. I think you only need to remove the reference to the env var.

Reply:
Yes, but it was designed to be able to run `run_l10n_extraction` multiple times, passing in a value for the app. Because that is no longer needed, we don't really need the function, so I figured we could clean up the file by moving the code outside of the function. My goal, at the end of a series of patches, is to not have any code that has anything to do with there being the possibility of multiple apps, so this change made sense to me. If you feel strongly about it I can undo this change.

Reply:
it is still a good practice to create functions in bash.

Reply:
Fair enough